src/quo2/components/record_audio/record_audio/view.cljs

```diff
@@ -135,7 +187,7 @@
                                   ;; GitHub issue [#14561]:
                                   ;; https://github.com/status-im/status-mobile/issues/14561
                                   (reset! clear-timeout
-                                    (utils/set-timeout
```

Comment:
is this because background timer don't work well with reanimated?

Reply:
Essentially I replaced these because of @flexsurfer comment in utils file (line 65) and also IDK how testable is background timer, jest provides really useful mocks for js default timers and worked really well for component tests (currently working on adding a few more tests). Also I saw other parts of the new codebase using js/setTimeout so I wasn't sure if we need to align to an specific approach for `setTimeout`s / `setInterval`s. I can replace it back if there's some performance reason and check how tests work.